### DISCUSSION TITLE
Gradient with background color opacity and custom percentage stop

### DISCUSSION THREAD

**Discussion by justin-harr** [tailwindlabs/tailwindcss#10705] [Help] [answered]
I want to replicate a custom linear-gradient like this with Tailwind:
```
linear-gradient(rgba(0, 0, 0, 0.5) 25%, rgba(0, 0, 0, 0) 100%);
```
I can replicate the custom opacity background color like this `from-black/[0.5]` and I can replicate the custom percentage color stop like this `from-[black_25%]`, but if I try to put it all together like this it doesn't work:
```
bg-gradient-to-r from-[black_25%]/[0.5]
```

Am I doing something wrong or is this just something Tailwind can't handle?

**Comment by wongjn** [comment, marked as answer]
The slash alpha notation only works for established color value. Here, you've got multiple values going on here, color and a percentage so the slash alpha notation won't work. However, you could supply a different color value that has the alpha value, like `from-[#00000080_25%]`, `from-[rgb(0_0_0/.5)_25%]`, `from-[rgba(0,0,0,.5)_25%]`, etc.

**Reply by justin-harr** [reply]
That worked perfectly thanks, I don't know why I forgot about using standard css alpha values to create the color transparency.

**Comment by Arunjenson** [comment]
can you help me with this
 background: linear-gradient(0deg, rgba(0, 0, 0, 0.50) 0%, rgba(0, 0, 0, 0.50) 100%), #363A43;

**Reply by wongjn** [reply]
`bg-gradient-to-t from-black/50 to-black/50 bg-[#363A43]`

**Comment by Topsymac** [comment]
can you help with this 
background: linear-gradient(180deg, #000014 -13.48%, #000 100%);

**Reply by wongjn** [reply]
`#14` isn't a valid color

**Reply by Topsymac** [reply]
it background: linear-gradient(180deg, 000014 -13.48%, #000 100%);

**Reply by Topsymac** [reply]
background: linear-gradient(180deg, # 000014 -13.48%, #000 100%);

**Reply by wongjn** [reply]
I assume you meant the hex code without a space, i.e. `#000014`? If so:

```lang-none
bg-gradient-to-b from-[#000014] from-[-13.48%] to-black
```

**Reply by Topsymac** [reply]
yes thank you so much 
It worked

**Comment by Topsymac** [comment]
can you help with this

background: linear-gradient(270deg, #FDF5ED 37.56%, rgba(253, 245, 237, 0.56) 86.46%, rgba(253, 245, 237, 0.41) 108.65%, rgba(253, 245, 237, 0.00) 201.31%);
